### ISSUE TITLE
tracy: various aggregates of device data for GUI

### ISSUE THREAD

**Issue by mo-tenstorrent** [tenstorrent/tt-metal#8900]
Currently device data is only displayed for all riscs on all cores on all devices. Op-report aggregations of device data needs to be added to make dealing with the data more manageable. 

Aggregate type 1:

per device compute risc-core activity (first risc-core - last risc-core) 

Aggregate type 2:

per device compute risc activity (first brisc-core - last brisc-core, first ncrisc-core to last ncrisc-core .....)

**Comment by mo-tenstorrent** [comment]
Blocked on https://github.com/tenstorrent/tt-metal/issues/14812

**Comment by mo-tenstorrent** [comment]
This might be not relevant or not as important with light profiler https://github.com/tenstorrent/tt-metal/issues/31589

Reducing to P3

**Comment by minaliuTT** [comment]
Per @mo-tenstorrent, this ticket is valid but low priority, and will be reviewed by the team when resources allow.
